### ISSUE TITLE
about ADNI dataset

### ISSUE THREAD

**Issue by Wongcheukwai** [VITA-Group/Self-PU#5]
Can you please tell me how to get rid.image_id.train.adni.npy, rid.image_id.test.adni.npy and adni_dx_suvr_clean.csv? Thx

**Comment by Wongcheukwai** [comment]
thank you for sharing in such a short time!!! I am applying for ADNI access, however, how to download the specific 822 training and 113 testing samples? I searched on the internet and people say its crashing all the time when they try to download even a small amount of images? Could you please share that too? I can delete this later. Appreciate it!

**Comment by chanyh0** [comment]
I am sorry to tell you that the (processed) ADNI data are non-shareable due to an NDA contract with an industry company since the company did the download and preprocessing process. So the best method to reproduce our result is to follow the reference provided in the paper and pre-process the data.

**Comment by Wongcheukwai** [comment]
Ok got it. Thank you ! Appreciate your help!

**Comment by laodar** [comment]
> thank you for sharing in such a short time!!! I am applying for ADNI access, however, how to download the specific 822 training and 113 testing samples? I searched on the internet and people say its crashing all the time when they try to download even a small amount of images? Could you please share that too? I can delete this later. Appreciate it!

Did you sucessuflly download the dataset? It seems that there are many data collections in the ADNI download page, e.g. [ADNI1:Baseline 3T], [ADNI1:Complete 1Yr] ... Which collections are required to contruct the training and test set?

**Comment by xxchenxx** [comment]
There seems to be a new ADNI dataset on Kaggle; maybe you can try that. https://www.kaggle.com/datasets/madhucharan/alzheimersdisease5classdatasetadni

**Comment by laodar** [comment]
> There seems to be a new ADNI dataset on Kaggle; maybe you can try that. https://www.kaggle.com/datasets/madhucharan/alzheimersdisease5classdatasetadni

Thanks, are the training and testing splits the same as your paper?

**Comment by xxchenxx** [comment]
Not really. I am just pointing out some possible alternatives (I have just found another: https://www.kaggle.com/tourist55/alzheimers-dataset-4-class-of-images)

As we stated before, our ADNI data are non-shareable due to an NDA contract with an industry company, since the company did the download and preprocessing process. Thanks for your interest but there is nothing more we can help at this moment, and the only solution seems to use these public datasets and set up your own benchmark.

**Comment by laodar** [comment]
> 

Ok, I got it, thank you.
